common/culture/traditions/00_ritual_traditions.txt

```diff
@@ -1393,7 +1423,24 @@ tradition_runestones = {
 
 	# Runestones are available in a slightly reduced form in the base title, so we let you access this tradition regardless of whether you have FP1 or not.
 	is_shown = {
-		like_north_germanic_group_trigger = yes
+		# Warcraft
+		# like_north_germanic_group_trigger = yes
+		
+		OR = {
+			AND = {
+				scope:character = { has_government = tribal_government }
+				NOR = {
+					any_in_list = { list = traits this = flag:ethos_courtly }
+					any_in_list = { list = traits this = flag:ethos_egalitarian }
+					any_in_list = { list = traits this = flag:ethos_spiritual }
+				}
+			}
+			any_culture_county = {
+				any_county_province = {
+					geographical_region = world_northrend
+				}
+			}
+		}
```

Comment:
Not sure about this. Do you want to represent totems like this?

Reply:
That's a neat idea. I hadn't thought about totems, but they work too. Do you think we should expand it to Tauren?

---

First, it should be established that `tradition_runestones` is the vanilla version of `tradition_fp1_northern_stories`. Currently, `tradition_runestones` is assigned only to Furbolg and where `tradition_fp1_northern_stories` is used, the fallback is `tradition_storytellers`. This is because in WoW, runestones are not a recurring motif for Vrykul and they have for the most part martial societies with oral storytelling traditions. The Stormheimr are the only Vrykulic culture who are overtly involved in the recording of lore and stories through written/carved means. Can you fact check me @Remerod? :)

That said, seeing as the Vrykul and Runestones are both derived from norse culture, I generalised their permissions across Northrend (We can do it by specific heritage or culture instead too) for players with NL that may wish to use them. I also accounted for the their existing descriptions, as follows:

>  tradition_fp1_northern_stories_name:0 "Northern Stories"
 tradition_fp1_northern_stories_desc:1 "Life is lived, told, and remembered through the lens of tales. What we say, and what people say about us, matters as much as what we do. We remember that, and it shows in our stories."

>  tradition_runestones_name:1 "Runestone Raisers"
 tradition_runestones_desc:1 "Death, conquest, the acquisition of rank — are these not the things that define life in the nobility? Where our people go, so go the markers of our lives' worth."

Runestones places more emphasis on the placement of the stones, whereas Northern Stories focuses on the tales themselves.

The other reason I generalised across Northrend was permit for Tuskarr stone raising. They are currently assigned `tradition_fp1_northern_stories` but players without NL dlc may wish to use `tradition_runestones`. From [WoWpedia](https://wowpedia.fandom.com/wiki/Tuskarr): 

[Tuskarr Stone Statues (There is a quest chain related to these statues in Dragonblight)
](https://static.wikia.nocookie.net/wowpedia/images/7/7e/Ss12-hires.jpg/revision/latest?cb=20070803211337)

Reply:
I would limit runestones only to vrykul or tuskkarr (the ones who use stones to tell story basically). Not region.

Reply:
How about totems as Runestones for Tauren and Furbolg? It's not a perfect match but other than 'Raise a Runestone' decision, the flavour text for both Runestones and Northern Stories are decent fits. A custom WC tradition may be added for this in the future.

This is how I have it set up right now.

**Runestones (Free version):**
> Death, conquest, the acquisition of rank — are these not the things that define life in the nobility? Where our people go, so go the markers of our lives' worth.

Is shown/can be picked only by cultures of Tauren, Odobenus, Vrykulic and Jalgaric heritage. Effectively, it is invisible to all of these cultures as they are already using Northern Stories with the exception of Furbolg. Furbolg use Runestones by default.

**BUT** it is also available like so:

```
			AND = {
				scope:character = { has_government = tribal_government }
				NOR = {
					any_in_list = { list = traits this = flag:ethos_courtly }
					any_in_list = { list = traits this = flag:ethos_egalitarian }
					any_in_list = { list = traits this = flag:ethos_spiritual }
				}
			}
```

Should I remove the above code and so limit it only to the aforementioned 4 cultural pillars?

---

**Northern Stories (DLC):**
> Life is lived, told, and remembered through the lens of tales. What we say, and what people say about us, matters as much as what we do. We remember that, and it shows in our stories.

Is shown only to cultures of Tauren[^1], Odobenus, Vrykulic and Jalgaric heritage.
Can only be selected if culture is in Northrend. 

Currently assigned to Frostborne Dwarves[^2], Tuskarr, Tauren, Highmountain and Stormheimr. Non-DLC fallback is Storytellers.

[^1]: I think Highmountain can keep Northern Stories, but maybe move Tauren to Runestones/Storytellers.
[^2]: Storm Peaks questlines about their history as titanic servitors of Loken vs Frost Giants + frozen battlefield at Thunderfall.

Reply:
> How about totems as Runestones for Tauren and Furbolg?

If you can adapt it. Also, check events related to the runestones.

Reply:
Adaptation left for #982. Still looking into runestone events.

Reply:
So yeah I would do it as a separate task and leave it to the northern races like vrykul and tuskarr for now.
Close the conversation, I think it's solved once you see it.